### ISSUE TITLE
[Feature request] Change the theme for the AppImage

### ISSUE THREAD

**Issue by italoghost** [RPCS3/rpcs3#16233]
### Quick summary

Hi! I would like the AppImage to follow the theme that I want, instead of it being locked down to the `gtk2` theme. 

### Details

I use GNOME and I have set all my QT apps to follow the settings that I have defined within `qt5ct` and `qt6ct` through the following environment variable: `QT_QPA_PLATFORMTHEME=qt5ct`.

Is it possible to change the file "linuxdeploy-plugin-qt-hook.sh", so it detects that there is another environment variable set, and use this one instead of `QT_QPA_PLATFORMTHEME=gtk2`?

**Comment by Samueru-sama** [comment]
Yes this is possible, you would need to copy the `styles` and  `platformthemes` plugins into the plugins dir of the appimage.

Then run `patchelf --set-rpath '$ORIGIN/../../lib'` on the libqt6ct libs.

With that said it doesn't look like rpcs3 plays nicely with qt6ct, the welcome screen looks good: 

![image](https://github.com/user-attachments/assets/556bb548-2060-4dfb-a64f-b724209ba60d)

But then I'm met with this: 

![image](https://github.com/user-attachments/assets/f48f8493-506e-4973-999c-8c37ae2e5487)

**Comment by italoghost** [comment]
Doesn't it depend on which font and configuration you are using?
